website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -62,6 +62,19 @@ from the Nomad client by setting [`set_environment_variables`][artifact_env].
 The use of filesystem isolation can be disabled in Client configuration by
 setting [`disable_filesystem_isolation`][artifact_fs_isolation].
 
+#### Datacenter Wildcards
+
+In Nomad 1.5.0, the
+[`datacenters`][/nomad/docs/job-specification/job#datacenters] field for a job
+accepts wildcards for multi-character matching. For example, `datacenters =
+["dc*"]` will match all datacenters that start with `"dc"`. The default value
+for `datacenters` is now `["*"]`, so the field can be omitted.
+
+The `*` character is no longer a legal character in the
+[`datacenter`][/nomad/docs/configuration#datacenter] field for an agent
+configuration. Before upgrading to Nomad 1.5.0, you should ensure that no agents
+have this character in their `datacenter` field name.
```

Comment:
Thinking about the upgrade path, would it make sense to update the jobs first? It's a weird case but if you have clients in `dc*` and `dc1` and a job with `datacenters = ["dc*"]` when you upgrade the clients (assuming the servers have been upgraded first as recommended), the replacement allocs may end up in `dc1`.

Reply:
Good catch.